common/components/componentsBySection/MyProjects/MyProjectCard.jsx

```diff
@@ -38,6 +38,11 @@ class MyProjectCard extends React.PureComponent<Props> {
                 </tr>
                 <tr>In Progress</tr>
               </td>
+              <td className="MyProjectCard-column">
+                <tr className="MyProjectCard-header">
+                  <a href={'?section=EditProject&id=' + this.props.project.id}>Edit</a>
```

Comment:
might be good to create some URLBuilder class that gets passed in a Section so that all these string literals don't break if we change our URL scheme?

Reply:
Done